tests/PerformanceHarness/README.md

```diff
@@ -747,6 +748,7 @@ Performance Test Basic Base:
                         Maximum size in megabytes http_plugin should use for processing http requests. -1 for unlimited. 429 error response when exceeded. (default: -1)
   --del-perf-logs       Whether to delete performance test specific logs. (default: False)
   --del-report          Whether to delete overarching performance run report. (default: False)
+  --save-state          Whether to save node state. (Warning: large log files)
```

Comment:
Maybe "(Warning: large disk usage)" <- state are really not "log" files.

Reply:
Addressed: [61b8464](https://github.com/AntelopeIO/leap/pull/1679/commits/61b8464ce58e74e74baf168ffd08120ea1d28e9d)